### PR TITLE
Register theheavy.is-a.dev

### PR DESCRIPTION
**Pull request by ArcticHub0**
Register theheavy.is-a.dev with CNAME record pointing to pynexus-pypi.github.io.

### CODE REVIEW

**Comment by PyNexus-PYPI** [comment]
Well, I made a mistake, the CNAME record should be: PyNexus-PYPI
Can you guys edit it before review? 
Thanks.

File: domains/theheavy.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "PyNexus-PYPI",
+           "email": "",
+           "discord": "1266113644643614803",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.Q0hi0eXIZoyV_qha54TNehZhpAbeSyzQI1OIT2wjQvArwviwVaqn4AKQj4LWMxDJFDu7KgAL-k336uR2UF26HMQ1h3BhR9t8xq_UdDJLrNv9HOyxBzLrVWZp3yveH__Cli62iKkBzJCu-0q2IbThvnZEhyyELgHBhXMot5eqWpm5vE_MGICc50lcz7vZNDPJTEGiie6bLdbvNPfxiXVugjhm7WW_tNoSheHSSOGr3asBTLFpl_46d-KGH07uKCc_wGgn9rz26qY0dwhaqTRDmq1lWn4_yuknefBHVw51vpk_C-7NA0lMp711CfPeqw0fKQT5n-QCSp4JfyYgXJkQhg.S6gJHj5xinUubawSe5bbEg.HR9U7iv7mJJAXQ1u5U88vuT0W9XbCd74QpxNl9Swn0QZ7DZs4foP8VbwLE7-L1EmEog8VSHnVWnrJO3SY06DZscCELHocxxPrOpe7cTcpomW3Sx4_fItRqIWV4OWl1Lr.qqnRh7IvZACgHMU5bwXM3g"
+        },
+    
+        "record": {
+            "CNAME": "pynexus-pypi.github.io/theheavy.github.io"
```
Comment (DEV-DIBSTER) [inline domains/theheavy.json:10]: CNAME must be a hostname.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/theheavy.json (hunk 1 above)
Comment (PyNexus-PYPI) [reply domains/theheavy.json:10]: Oh. Okay. Give me a moment.

Comment (PyNexus-PYPI) [reply domains/theheavy.json]: Is that now looking good?

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**